src/Foundation/NSObject2.cs

```diff
@@ -51,11 +51,18 @@
 #endif
 
 namespace Foundation {
+
+#if NET
+	public enum NSObjectFlag {
```

Comment:
Just based on the name, should this be a flag enum?

Reply:
(by design) it only has one value... 😄 
TBH I would have changed the name too - if it had not been a much larger change (and the fact I did not think of anything else)